src/annotator/plugin/bucket-bar.coffee

```diff
@@ -251,7 +253,7 @@ module.exports = class BucketBar extends Plugin
           scrollToClosest(@buckets[bucket], 'down')
         else
           annotations = (anchor.annotation for anchor in @buckets[bucket])
-          annotator.selectAnnotations annotations,
+          @annotator.selectAnnotations annotations,
```

Comment:
This is the actual bug fix. The value assigned to `this.annotator` / `@annotator` used to be assigned to a global variable `annotator`.